### PR TITLE
Update package, language, core and compiler tool versions

### PR DESCRIPTION
**Pull request by KevinRansom**
Update package, tool and language version numbers.

FSharp.Core package ==>  4.6.3
Microsoft.FSharp.Compiler ==> 10.4.3

FSharp tools banner  ==>  10.4.3

Please note:  Samples package references need to reflect already shipped versions.

### CODE REVIEW

File: fcs/samples/Tokenizer/Tokenizer.fsproj [hunk 1]
```diff
@@ -13,7 +13,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="4.6.2" />
-    <PackageReference Include="FSharp.Core" Version="4.5.2" />
```
Comment (cartermp) [inline fcs/samples/Tokenizer/Tokenizer.fsproj]: Why is 4.5.2 being removed here?

Comment (KevinRansom) [reply fcs/samples/Tokenizer/Tokenizer.fsproj]: As you can see there are two package references.  Clearly one is redundant.

Comment (KevinRansom) [reply fcs/samples/Tokenizer/Tokenizer.fsproj]: As for bumping the fsharp.core version 4.6.2 has already shipped, this is in preparation for shipping the next one.  Which will likely happen when we ship a new compiler, in Dev 16.1

Comment (cartermp) [reply fcs/samples/Tokenizer/Tokenizer.fsproj]: 4.5.2 removal makes sense here, but is it necessary (from a build process) to rev 4.6.2 with a new compiler if we're not shipping changes?

File: fcs/samples/EditorService/EditorService.fsproj [hunk 2]
```diff
@@ -13,7 +13,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="4.6.2" />
-    <PackageReference Include="FSharp.Core" Version="4.5.2" />
     <PackageReference Include="FSharp.Core" Version="4.1.19" />
```
Comment (brettfo) [inline fcs/samples/EditorService/EditorService.fsproj]: I suspect FSharp.Core 4.1.19 appeared as a merge error and can be cleaned up, too.  Same for other projects.

File: fcs/samples/Tokenizer/Tokenizer.fsproj (hunk 1 above)
Comment (KevinRansom) [reply fcs/samples/Tokenizer/Tokenizer.fsproj]: @cartermp, this is in preparation for shipping the next FSharp.Core package.  Whether we ship it on a particular date or note is a decision for then.

File: fcs/samples/EditorService/EditorService.fsproj (hunk 2 above)
Comment (KevinRansom) [reply fcs/samples/EditorService/EditorService.fsproj]: Don submitted a pr that addresses this.

**Pull request merged**